api/v1alpha1/httproute_types.go

```diff
@@ -115,15 +116,21 @@ type HTTPRouteMatch struct {
 	// Default: "/"
 	Path *string `json:"path" protobuf:"bytes,2,opt,name=path"`
 
-	// HeaderType defines the semantics of the `Header` matcher.
+	// HeaderMatchType defines the semantics of the `Header` matcher.
+	//
+	// Support: core (Exact)
+	// Support: custom (ImplementationSpecific)
+	//
+	// Default: "Exact"
 	//
 	// +optional
-	HeaderType *string `json:"headerType" protobuf:"bytes,3,opt,name=headerType"`
-	// Header are the Header matches as interpreted via
-	// HeaderType.
+	HeaderMatchType *string `json:"headerType" protobuf:"bytes,3,opt,name=headerType"`
+	// Headers are the HTTP Headers to match as interpreted via
+	// HeaderMatchType. Mutliple headers are ANDed together, meaning, a request
+	// must contain all the headers specified inorder to select this route.
```

Comment:
Maybe need a space here
```suggestion
	// must contain all the headers specified in order to select this route.
```